03week/ticTacToe.js

```diff
@@ -23,32 +46,91 @@ function printBoard() {
   console.log('2 ' + board[2].join(' | '));
 }
 
-function horizontalWin() {
-  // Your code here
+const isValidInput = (row, column) => {
+  const validInputs = [0, 1, 2];
+  const isValid = validInputs.indexOf(row) > -1 || validInputs.indexOf(column) > -1
```

Comment:
This line your logic only checks if one of the parameters is greater than -1 but not the other.  resulting in a error

Reply:
just changed it to && instead of ||